cmake/compiler_utils.cmake

```diff
@@ -56,4 +56,31 @@ function(get_muparserx_source_code)
                 Please download MuparserX library from https://github.com/beltoforion/muparserx.git \
                 and checkout latest stable release")
     endif()
-endfunction()
\ No newline at end of file
+endfunction()
+
+function(check_compiler_cpp11_abi)
+    # Consider also if -D_GLIBCXX_USE_CXX11_ABI has been passed as flag
```

Comment:
Can we comment here why all this is needed? With the link to the original issue (bugzilla) and a short description... Our "old me" and newcomers to the project will thank some more clarification :)

Reply:
Totally agree!